spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/HyperConformance.scala

```diff
@@ -90,18 +90,20 @@ class HyperConformance (implicit cmd: ConformanceConfig,
  *
  * In order to use it in hyperdrive the component needs to be configured in 'ingestion.properties' as follows:
  * {{{
- * transformer.hyperconformance.menas.rest.uri=http://localhost:8080
- * transformer.hyperconformance.dataset.name=example
- * transformer.hyperconformance.dataset.version=1
- * transformer.hyperconformance.report.date=2020-01-29
- * transformer.hyperconformance.report.version=1
- * transformer.hyperconformance.event.timestamp.column=EV_TIME
+ * component.transformer.id.1=[hyperconformance]
```

Comment:
What does the enclosing in brackets do/mean here? Optional?

Reply:
It doesn't mean anything per se. Here, whatever you define in `component.transformer.id.1` is used in the key of other properties. In this sense, the brackets show that this part of the key is dynamic and user-defined. But you could also not use brackets, here they are just characters that stand out. I didn't think it could be interpreted as optional.

So, you could also write
```
component.transformer.id.1=asdf
component.transformer.class.asdf=...
transformer.asdf.means.rest.uri=...
```